stacks/client_stack.py

```diff
@@ -140,8 +138,10 @@ def apigw(self):
             # authorizer=authorizer_iam
         )
         
-        self.apigw_full_invoke_url = f'{self.http_api.url}{self.path}'
+        self.apigw_full_invoke_url = f'{self.http_api.url[:-1]}{self.path}' # remove duplicate slash
```

Comment:
A more robust way of doing this is the following:

```
In [1]: from os import path

In [5]: path.join("https://google.com/some", "path".strip('/'))
Out[5]: 'https://google.com/some/path'

In [6]: path.join("https://google.com/some", "///path/".strip('/'))
Out[6]: 'https://google.com/some/path'

In [7]: path.join("https://google.com/some////", "///path/".strip('/'))
Out[7]: 'https://google.com/some////path'

In [8]: path.join("https://google.com/some////".rstrip("/"), "///path/".strip('/'))
Out[8]: 'https://google.com/some/path'
```

It allows for both the base and the path to either have slashes or not, and uses only one slash in the final result. Note that I added a `.rstrip()` call in the last example, but that isn't totally necessary.